rosdep/base.yaml

```
@@ -3593,6 +3593,20 @@ libdw-dev:
       packages: [elfutils]
   rhel: [elfutils-devel]
   ubuntu: [libdw-dev]
+libdwarf-dev:
+  alpine: [libdwarf-dev]
+  arch: [libdwarf]
+  debian: [libdwarf-dev]
+  fedora: [libdwarf-devel]
+  gentoo: [dev-libs/libdwarf]
+  nixos: [libdwarf]
+  openembedded: [libdwarf]
+  opensuse: [libdwarf-devel]
+  osx:
+    homebrew:
+      packages: [dwarfutils]
+  rhel: [libdwarf-devel]
+  ubuntu: [libdwarf]
```

```suggestion
  ubuntu: [libdwarf-dev]
```
This looks like an invalid key for ubuntu. There is libdwarf1 but this is a dev key and uses the dev key for all the other rules. So should probably here too. 

https://packages.ubuntu.com/noble/libdwarf-dev

---

Ouch, thanks!